### Same "cl:variables:star-compile-print-star.diff"
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:variables:star-compile-verbose-star.txt	2017-04-27 21:40:30.000000000 +0100
+++ saved-pages-clus/cl:variables:star-compile-verbose-star.txt	2017-04-27 21:40:30.000000000 +0100
@@ -1,19 +1,47 @@
 Variable *COMPILE-PRINT*, *COMPILE-VERBOSE* 
  
 Value Type
  
 a generalized boolean. 
 Initial Value
  
 implementation-dependent. 
 Description
  
-The value of *compile-print* is the default value of the print argument to compile-file. The value of *compile-verbose* is the default value of the verbose argument to compile-file. 
-Examples None.
  
-Affected By None.
+The value of *compile-print* is the default value of the print argument to compile-file. The value of *compile-verbose* is the default value of verbose argument to compile-file.
+Examples
+None.
+Affected By
+None.
  
 See Also
  
-compile-file 
-Notes None.
\ No newline at end of file
+ Function COMPILE-FILE
+Notes
+None.
+\
\ No newline at end of file
